libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -454,6 +454,23 @@ public static async Task<BatchFailedEntriesResponse> GetPagedFailedEntriesAsync(
             }
         }
 
+        /// <summary>
+        /// Cancels a batch operation by its id.
+        /// </summary>
+        /// <param name="turnContext"> Turn context. </param>
+        /// <param name="operationId"> The id of the operation to cancel. </param>
+        /// <param name="cancellationToken"> The cancellation token. </param>
+        /// <returns>The response state code.</returns>
```

Comment:
We are not returning any state. The comment should be:
```suggestion
        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
```